api/client/samples/prevented_plant/prevent-plant-model-overview.md

```diff
@@ -0,0 +1,51 @@
+<p align="center"><img width=8% src="https://gro-intelligence.com/images/logo.jpg"></p>
+
+# Prevent Plant Model for US Corn and Soybeans Using Gro
+
+## Goal & Process
+Basic Model Description: The prevent plant model endeavors to forecast prevent plant area for corn and soybeans at the county level. Using datasets in Gro, the model will examine the overall wetness across all Corn and Soybean Belt counties using soil moisture, rainfall, planting progress reports, and historical prevented plant area. 
+
+Prevented plant is the area farmers intended to plant, but were unable to, typically because of extreme weather. Farmers are entitled to a payout from revenue-insurance contracts if the ground remains unplanted (for more information see https://www.rma.usda.gov/Topics/Prevented-Planting)
+
+## Model Inputs
+__Corn Prevent Plant Model Inputs:__ 
+[SMOS Soil Moisture](https://app.gro-intelligence.com/dictionary/sources/43)
+* __Model Specific Data:__ Weekly averages for soil moisture by county for the US Corn Belt. 
+[Tropical Rainfall Measuring Mission (TRMM)](https://app.gro-intelligence.com/dictionary/sources/35)
+* __Model Specific Data:__ Weekly cumulative precipitation by county for the US Corn Belt. 
+[USDA NASS](https://app.gro-intelligence.com/dictionary/sources/25)
+* __Model Specific Data:__ The weekly progress report for corn planted for the US Corn Belt. The report measures area planted as a percentage of the intended area. Data is measured on a state level and applied to each county in the model. 
+[USDA FSA](https://app.gro-intelligence.com/dictionary/sources/100)
+* __Model Specific Data:__ “Area prevented, non-irrigated” data to determine acres that have not been planted in the current crop year due to an inability to plant the crop. The final prevent plant number, released by the FSA in January, is what the model is predicting. 
+Latitude and Longitude 
+* County level locations for relevant counties being modeled, which can be found using an API function in Gro.
+
+__Soybean Prevent Plant Model Inputs:__ 
+*Note, in addition to inputs used for the corn model, the soybean model includes the following:*

```

Comment:
Similar to SMOS above, this note is not bulleted and so shows up on the same line as "Soybean Prevent Plant Model Inputs"

Reply:
Perhaps you meant to surround this line with blank lines to get it to show up, unbulleted, between the title and the list